systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -345,8 +357,8 @@ public void testCustomAndUpdatedValues() {
      * Test sending messages over plain transport, without auth
      */
     @Test
-    @JUnitGroup(name = "acceptance")
-    public void testSendMessagesPlainAnonymous() throws InterruptedException {
+    @Tag("regression")
```

Comment:
Why did you moved so many tests from regression to acceptance? Did you made the test times so much faster to make them fit the time limit?

Reply:
I did wrong refactor there. There will be no changes in the tag from regression to acceptance. I already change it cause Travis failures.